.github/workflows/publish-operators-for-e2e-tests.yml

```diff
@@ -64,12 +64,12 @@ jobs:
         python-version: '3.x'
 
     - name: Prepare tools
-      uses: codeready-toolchain/toolchain-cicd/prepare-tools-action@master
+      uses: xcoulon/toolchain-cicd/prepare-tools-action@d426c7b
 
     # Execute composite action using values from PR event
     # Is executed only for PR events - in that case the comment field is empty
     - name: Publish current operator bundles for host & member based on PR event
-      uses: codeready-toolchain/toolchain-cicd/publish-operators-for-e2e-tests@master
+      uses: xcoulon/toolchain-cicd/publish-operators-for-e2e-tests@d426c7b
```

Comment:
Let's revert these changes

Reply:
reverted in https://github.com/codeready-toolchain/toolchain-e2e/commit/7e01e2fe1f9159f8a27001e50f1c5e881390ad10